.travis.yml

```diff
@@ -48,9 +47,13 @@ matrix:
       env:
         - SWIFT_VERSION=5.0.3
     - <<: *swiftpm_linux
-      name: SwiftPM / Linux / Swift 5.1 Development
+      name: SwiftPM / Linux / Swift 5.1.4
       env:
-        - SWIFT_VERSION=5.1-DEVELOPMENT-SNAPSHOT-2019-09-09-a
-
+        - SWIFT_VERSION=5.1.4
+    - <<: *swiftpm_linux
+      name: SwiftPM / Linux / Swift 5.2 Development
+      env:
+        - SWIFT_VERSION=5.2-DEVELOPMENT-SNAPSHOT-2020-02-09-a
```

Comment:
Swift 5.2 does not support Ubuntu 14.04 anymore.

https://github.com/kylef/swiftenv-api/blob/master/versions/5/5.2-DEVELOPMENT-SNAPSHOT/2020-02-09-a.yaml